src/sdk/util/passphrase.ts

```diff
@@ -0,0 +1,39 @@
+import { topPasswords } from '../topPassphrases';
+import { randomBytes } from './crypto';
+
+export function generateRandomKeywords(): string[] {
+    const randomIndices: number[] = [];
+    const maxIndex = topPasswords.length;
+
+    while (randomIndices.length < 6) {
+        // Generate a random 4-byte number
+        const randomBytesArray = randomBytes(4);
+        const randomIndex = byteArrayToNumber(randomBytesArray) % maxIndex;
+
+        if (!randomIndices.includes(randomIndex)) {
+            randomIndices.push(randomIndex);
+        }
+    }
+
+    const randomKeywords: string[] = randomIndices.map((index) => topPasswords[index]);
+
+    return randomKeywords;
+}
+
+export function generateAutoSuggestions(inputString: string): string[] {
```

Comment:
Add some TSDoc to explain the input and output (That will make it clear that this returns maximum 4 words)